pipeline-scripts/buildlib.groovy

```diff
@@ -1453,55 +1453,53 @@ def buildBuildingPlashet(version, release, el_major, include_embargoed, auto_sig
     /**
      * The yum repos that plashet creates are very lightweight because
      * it only symlinks out to the desired RPMs on buildvm's /mnt/redhat
-     * share. We now want to copy the new repo out to rcm-guest. The
-     * good news is that we can keep it lightweight! rcm-guest has the
+     * share. We now want to copy the new repo out to ocp-artifacts. The
+     * good news is that we can keep it lightweight! ocp-artifacts has the
      * exact same share path. The symlinks plashet created can be copied
      * as symlinks. Note that if you copy the puddle to a system without
      * these links, you should use rsync --copy-links which will transfer
      * the linked file's content to the destination filename.
      *
-     * Now.. let's copy to rcm-guest! Why? Because when we build images in
+     * Now.. let's copy to ocp-artifacts! Why? Because when we build images in
      * brew, OSBS will only let the Dockerfile's yum invocations
      * access certain remote locations. One of those whitelisted locations
-     * is rcm-guest, so copy our lightweight repo there.
+     * is ocp-artifacts, so copy our lightweight repo there.
      *
-     * ocp-build is a user established for us on the rcm-guest host by
-     * Red Hat PnT devops. See /home/jenkins/.ssh.config.
      */
 
     // During an image build, doozer will provide repo files pointing back to a directory named
-    // 'building' in this rcm-guest directory. Before creating 'building', let's get the
+    // 'building' in this ocp-artifacts directory. Before creating 'building', let's get the
     // repo over there.
-    def destBaseDir = "/mnt/rcm-guest/puddles/RHAOS/plashets/${major_minor}"
+    def destBaseDir = "/mnt/data/pub/RHOCP/plashets/${major_minor}"
```

Comment:
If you look at `buildlib.build_plashets` closely, `./hacks/plashet/build-plashet.py` is to create plashet repos on ocp-artifiacts while this function is untouched to continue to do for rcm-guest.

So I would like to delete this function after 3.11 is migrated. Also I would like to avoid to run command directly on ocp-artifacts if possible.

Reply:
I had noticed that, but I figured I would apply the least changes as possible. When 3.11 will go (and one day it will), we can simply remove ocp3 job and related functions

Reply:
I am ok with doing a minimum change.